exe/wallet/jormungandr/Main.hs

```diff
@@ -345,7 +347,7 @@ cmdServe = command "serve" $ info (helper <*> cmd) $ mempty
         withDBLayer
             :: CM.Configuration
             -> Trace IO Text
-            -> (DBLayer IO s t -> IO a)
+            -> (DBLayer IO s t SeqKey -> IO a)
```

Comment:
Minor note but, this should / could be `k`, as aliased above :point_up:

Reply:
fixed in https://github.com/input-output-hk/cardano-wallet/pull/597/commits/d7577071415476ffcd23e89749d9391e37bff24c